examples/tests/sdk_tests/fs/path.main.w

```diff
@@ -0,0 +1,36 @@
+bring fs;
+
+let from = "/a/b/c/d";
+let to = "/a/b/e/f";
+
+let var result = fs.join(from, to);
+assert(result == "/a/b/c/d/a/b/e/f");
+
+result = fs.relative(from ,to);
+assert(result == "../../e/f");
+
+result = fs.dirname(from);
+assert(result == "/a/b/c");
+
+result = fs.basename(from);
+assert(result == "d");
+
+result = fs.resolve(from, to);
+assert(result == "/a/b/e/f");
+
+test "inflight path conversion" {
+    let var result = fs.join(from, to);
```

Comment:
@MarkMcCulloh @Chriscbr @yoav-steinberg any thoughts on how we can avoid the duplicated test? It's related to phase independence

Reply:
IMO the most straightforward solution for the duplicate code is to write the test logic in an `unphased` function and then call the function in both preflight and inflight.

https://github.com/winglang/wing/pull/1711